diff.py

```diff
@@ -445,7 +447,9 @@ def create_project_settings(settings: Dict[str, Any]) -> ProjectSettings:
         objdump_executable=get_objdump_executable(settings.get("objdump_executable")),
         objdump_flags=settings.get("objdump_flags", []),
         map_format=settings.get("map_format", "gnu"),
-        mw_build_dir=settings.get("mw_build_dir", "build/"),
+        ms_map_address_offset=settings.get("ms_map_address_offset", 0),
+        ms_ignore_missing_objfile=settings.get("ms_ignore_missing_objfile", True),
+        build_dir=settings.get("build_dir", "build/"),
```

Comment:
```suggestion
        build_dir=settings.get("build_dir", settings.get("mw_build_dir", "build/")),
```
to avoid hassle for mwcc decomp projects